resources/functions/text-category.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: serverless.kyma-project.io/v1alpha1
+kind: Function
+metadata:
+  name: text-category
+specs:
+  deps: |-
+    {
+      "name": "text-category",
+      "version": "1.0.0",
+      "dependencies": {
+        "node-fetch": "2.6.1",
+        "@google-cloud/language":"4.3.0",
+        "nanoid": "3.1.25"
+      }
+    }
+  env:
+    - name: API_CONFIG
+      valueFrom:
+        configMapKeyRef:
+          key: config.json
+          name: showcase-frontend-config
+    - name: GCP_EMAIL
+      valueFrom:
+        secretKeyRef:
+          key: GCP_EMAIL
+          name: kyma-showcase-secret
+    - name: GCP_API_KEY
+      valueFrom:
+        secretKeyRef:
+          key: GCP_API_KEY
+          name: kyma-showcase-secret
+  runtime: nodejs14
+  source: |-
+    module.exports = {
+      main: async function (event, context) {
+        async function getText(id) {
+          console.log("Getting text from DB");
+          const fetch = require('node-fetch');
+          const URL = JSON.parse(process.env.API_CONFIG).API_URL + '/' + id;
+          const response = await fetch(URL);
+          const data = await response.json();
+          const content = data.gcp.map(JSON.parse);
+          const font = content?.find(obj => Object.keys(obj).includes('font'))?.font || [];
+          console.log("Text loaded");
+          return font;
+        }
+
+      async function getTextCategory(text) {
+        console.log("Getting text category");
+        const language = require('@google-cloud/language');
+        const options = {
+            credentials: {
+              client_email: process.env.GCP_EMAIL,
+              private_key: process.env.GCP_API_KEY.replace(/\\n/gm, '\n'),
+            },
+        };
+
+        const client = new language.LanguageServiceClient(options);
+        const document = {
+          content: text,
+          type: 'PLAIN_TEXT',
+        };
+        const [classification] = await client.classifyText({document});
+        const results = classification.categories.map(category => ({
+          name: category.name,
+          confidence: category.confidence,
+        }));
```

Comment:
![image](https://user-images.githubusercontent.com/46759954/132672141-f8f05cc4-116e-40d1-af1e-bbe3c16d4d62.png)
Idk why, but it is a very very bad response to DB.